src/tracer.zig

```diff
@@ -423,15 +424,15 @@ const TracerTracy = struct {
         comptime format: []const u8,
         args: anytype,
     ) void {
-        const level_txt = comptime level.asText();
+        const level_text = comptime level.asText();
         const prefix = if (scope == .default) ": " else "(" ++ @tagName(scope) ++ "): ";
         const message = std.fmt.bufPrint(
             &print_buffer,
-            level_txt ++ prefix ++ format,
+            level_text ++ prefix ++ format,
             args,
         ) catch message: {
             const dots = "...";
-            std.mem.copy(u8, print_buffer[print_buffer.len - dots.len ..], dots);
+            util.copy_disjoint(u8, print_buffer[print_buffer.len - dots.len ..], dots);
```

Comment:
You also need to pass either `.exact` or `.inexact` as the first argument (these are exact I think).

Reply:
Uh, getting bitten by the fact that the backend code is not type-checked if not enabled.